src/main/java/io/jenkins/plugins/util/AgentFileVisitor.java

```diff
@@ -42,30 +42,32 @@
     private final String filePattern;
     private final String encoding;
     private final boolean followSymbolicLinks;
+    private final boolean errorOnEmptyFiles;
     private final FileSystemFacade fileSystemFacade;
+    private static final String EMPTY_FILE = "Skipping file '%s' because it's empty";
 
     /**
      * Creates a new instance of {@link AgentFileVisitor}.
-     *
      * @param filePattern
      *         ant file-set pattern to scan for files to parse
      * @param encoding
      *         encoding of the files to parse
      * @param followSymbolicLinks
-     *         if the scanner should traverse symbolic links
```

Comment:
Why did you delete the parameter description?

Reply:
Resolved thanks for the comment